#! /bin/sh /usr/share/dpatch/dpatch-run
## 001_emptyfile.dpatch by Kevin Coyner <kcoyner@debian.org>
##
## DP: allows empty css files to be generated, which are actually valid.
## DP: thanks to Bastian Kleineidam <calvin@debian.org>

@DPATCH@
diff -urNad csstidy-1.4~/csstidy/print_css.cpp csstidy-1.4/csstidy/print_css.cpp
--- csstidy-1.4~/csstidy/print_css.cpp	2007-07-10 10:20:52.000000000 -0400
+++ csstidy-1.4/csstidy/print_css.cpp	2007-09-21 22:13:06.000000000 -0400
@@ -78,8 +78,7 @@
 {
 	if(css.empty() && charset == "" && namesp == "" && import.empty() && csstokens.empty())
 	{
-		if(!settings["silent"]) cout << "Invalid CSS!" << endl;
-		return;
+        if(!settings["silent"]) cout << "Warning: empty CSS output!" << endl;
 	}
 
 	ofstream file_output;
